### ISSUE TITLE
Version 0.4.2 is entirely broken in Firefox

### ISSUE THREAD

**Issue by SayanthD** [kairi003/Get-cookies.txt-LOCALLY#29]
Nothing is working in Firefox and since it [doesn't support](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/background#browser_compatibility) "background.service_worker" with Manifest v3, I can't manually load this extension from the source at the latest commit.  But its fine, the real problem is that button-functions aren't working in here.

Using Firefox stable: 111.0.1 (64-bit)
Extension version 0.4.2
Installed from `https://addons.mozilla.org/en-US/firefox/addon/get-cookies-txt-locally/`

Logs:

- Copy
```
Uncaught (in promise) ReferenceError: ClipboardItem is not defined
    setClipboard moz-extension://5e5e85ef-387a-43e8-981d-ee361b4bc602/popup.js:65
    <anonymous> moz-extension://5e5e85ef-387a-43e8-981d-ee361b4bc602/popup.js:122
    async* moz-extension://5e5e85ef-387a-43e8-981d-ee361b4bc602/popup.js:119
```

- Export As
```
Uncaught (in promise) Error: Type error for parameter options (Error processing url: Error: Access denied for URL data:text/plain,%23%20Netscape%20HTTP%20Cookie%20File%0A%23%20http%3A%2F%2Fcurl.haxx.se%2Frfc%2Fcookie_spec.html%0A%23%20This%20is%20a%20generated%20file!%20%20Do%20not%20edit.%0A) for downloads.download.
    save moz-extension://5e5e85ef-387a-43e8-981d-ee361b4bc602/popup.js:55
    <anonymous> moz-extension://5e5e85ef-387a-43e8-981d-ee361b4bc602/popup.js:116
    async* moz-extension://5e5e85ef-387a-43e8-981d-ee361b4bc602/popup.js:112
```

- Export
```
Uncaught (in promise) Error: Type error for parameter options (Error processing url: Error: Access denied for URL data:text/plain,%23%20Netscape%20HTTP%20Cookie%20File%0A%23%20http%3A%2F%2Fcurl.haxx.se%2Frfc%2Fcookie_spec.html%0A%23%20This%20is%20a%20generated%20file!%20%20Do%20not%20edit.%0A) for downloads.download.
    save moz-extension://5e5e85ef-387a-43e8-981d-ee361b4bc602/popup.js:55
    <anonymous> moz-extension://5e5e85ef-387a-43e8-981d-ee361b4bc602/popup.js:109
    async* moz-extension://5e5e85ef-387a-43e8-981d-ee361b4bc602/popup.js:105
```

- Export All Cookies
```
Uncaught (in promise) Error: Type error for parameter options (Error processing url: Error: Access denied for URL data:text/plain,%23%20Netscape%20HTTP%20Cookie%20File%0A%23%20http%3A%2F%2Fcurl.haxx.se%2Frfc%2Fcookie_spec.html%0A%23%20This%20is%20a%20generated%20file!%20%20Do%20not%20edit.%0A%0A........
    save moz-extension://5e5e85ef-387a-43e8-981d-ee361b4bc602/popup.js:55
    <anonymous> moz-extension://5e5e85ef-387a-43e8-981d-ee361b4bc602/popup.js:128
    async* moz-extension://5e5e85ef-387a-43e8-981d-ee361b4bc602/popup.js:125
```

**Comment by kairi003** [comment]
The source code in the master branch does not support Firefox.
The firefox branch contains diffs for it.

We have confirmed that the version currently available on AMO works with 111.0.1.

![Screenshot from 2023-04-11 21-55-51](https://user-images.githubusercontent.com/24368162/231169011-7db09766-554e-49fe-ab78-325e28b38826.png)

**Comment by kairi003** [comment]
v0.4.3, which resolves this issue, is now published on AMO.
Thank you!
